### PR TITLE
added tests for home page and dividend calculator page

### PR DESCRIPTION
**Pull request by UserOk007**
@Olha-Vasylets Hi! Attaching a screenshot with test results: 
image

### CODE REVIEW

File: cypress/e2e/dividendCalculator.cy.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        cy.visit(urls.dividendCalculatorUAT);
+        cy.wait(5000);
+        cy.get(selectors.cookiesPopUp.acceptButton).should('be.visible').click();
+        cy.wait(3000);
+    })
+
+    it.only('select location, go to step 2, check error if user doesn"t agree conditions, go to page 3, verify error messages are shown if user submits form with empty fields', () => {
+        cy.get(selectors.scriptDividend.locationDropDown).select(1);
+        cy.get('button').contains(testData.dividendCalculator.submitButton).should('not.have.class', selectors.scriptDividend.disabledButtonClass).click();
+        cy.get(selectors.scriptDividend.buttonTextClass).contains(testData.dividendCalculator.dontAgreeButton).click({force: true});
```
Comment (Olha-Vasylets) [inline cypress/e2e/dividendCalculator.cy.js:21]: {force: true} why is it needed here?

Comment (UserOk007) [reply cypress/e2e/dividendCalculator.cy.js:21]: The system showed an error and it didn't click on the button. There was a recommendation in cypress running tests area to add {force: true}. After adding, it started to work

**Review by Olha-Vasylets — CHANGES_REQUESTED** [review]
Please correct all the points we discussed. And remember, you're a rock anyway :)

File: cypress/e2e/homePage.cy.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    });
+
+    it('switch the country and language from the footer area', () => {
+        cy.get(selectors.homePage.changeLocationButtonInFooter).click();
+        cy.get(selectors.homePage.countryValueInCountrySelectionMenu).should('have.length', testData.homePage.quantityOfCountries);
+        cy.get(selectors.homePage.countryValueInCountrySelectionMenu).eq(27).invoke('removeAttr', 'target').click();
+        cy.url().should('include', testData.homePage.partialUrlForFranceFrench);
+        cy.get(selectors.homePage.currentCountryLanguageInFooter).eq(0).should('contain', testData.homePage.currentCountry);
+        cy.get(selectors.homePage.currentCountryLanguageInFooter).eq(1).should('contain', testData.homePage.currentLanguage);
+    });
```
Comment (Olha-Vasylets) [inline cypress/e2e/homePage.cy.js:53]: **let's try to add a slightly more readable structure?** 

```
    cy.get(selectors.homePage.countryValueInCountrySelectionMenu)
        .eq(27)
        .invoke('removeAttr', 'target')
        .click();

    cy.get(selectors.homePage.currentCountryLanguageInFooter)
        .eq(0)
        .should('contain', testData.homePage.currentCountry);
    cy.get(selectors.homePage.currentCountryLanguageInFooter)
        .eq(1)
        .should('contain', testData.homePage.currentLanguage);

```
 _:) yes, it is longer but easier to read/watch_

**Review by Olha-Vasylets — CHANGES_REQUESTED** [review]
please break into lines

**Review by UserOk007 — COMMENTED** [review]
@Olha-Vasylets  Hi! I committed changes to the previous branch. Using chatGPT I tried to cancel my commit to add it to the new branch but it was not successful. All my last changes are in this commit. Please review.

**Review by Olha-Vasylets — APPROVED** [review]

**Pull request merged**